src/base/projects/Tree/Folder.svelte

```diff
@@ -1,20 +1,28 @@
 <script lang="ts" strictEvents>
   import { createEventDispatcher } from "svelte";
 
+  import File from "./File.svelte";
   import Loading from "@app/Loading.svelte";
   import type { Tree } from "@app/project";
   import { ObjectType } from "@app/project";
+  import { browserErrorStore } from "../Browser.svelte";
 
-  import File from "./File.svelte";
-
-  export let fetchTree: (path: string) => Promise<Tree>;
+  export let fetchTree: (path: string) => Promise<Tree | undefined>;
   export let name: string;
   export let prefix: string;
   export let currentPath: string;
   export let loadingPath: string | null = null;
 
   let expanded = currentPath.indexOf(prefix) === 0;
-  let tree: Promise<Tree> | null = expanded ? fetchTree(prefix) : null;
+  let tree: Promise<Tree | undefined> | null = expanded
```

Comment:
Why do we mix undefined and null here?

Reply:
Has been rewritten to only use `undefined`